frontend/packages/helm-plugin/src/components/list-page/HelmTabbedPage.tsx

```diff
@@ -62,7 +63,7 @@ export const PageContents: React.FC<HelmTabbedPageProps> = (props) => {
 
   const menuActions: MenuActions = {
     helmRelease: {
-      label: t('helm-plugin~Helm Release'),
+      label: isHelmVisible ? t('helm-plugin~Helm Release') : null,
       onSelection: () => `/catalog/ns/${namespace}?catalogType=HelmChart`,
     },
```

Comment:
If the flag is false and we don't want to add an `Action`, we should not add it completely instead of setting the label to null.

Reply:
If the label is null then that action will not be added.